src/styles/foundation/spacing.scss

```diff
@@ -1,13 +1,6 @@
-$unit: 4px;
+@import '../polaris-tokens/spacing.spacing-map';
```

Comment:
Cross linking to this file, in case someone needs context:

https://github.com/Shopify/polaris-tokens/blob/4564a255e2ff5cfda0119483e43dc0d55e8c0094/dist/spacing.spacing-map.scss